lua/render-markdown/handler/markdown.lua

```diff
@@ -222,6 +222,30 @@ M.render_code = function(buf, info)
         },
     }
     list.add(marks, background_mark)
+
+    if code.width == 'block' then
+        -- overwrite anything beyond block width + right_pad with Normal
+        local width = 0
+        local lines = vim.api.nvim_buf_get_lines(buf, start_row, end_row, false)
+        for _, line in ipairs(lines) do
+            width = math.max(width, str.width(line))
+        end
+        local pad = str.pad('', vim.o.columns * 2 - width)
```

Comment:
I assume the point of the `* 2` here is to guarantee the padding fills the column width? In which case can we do:

```lua
local pad = str.pad('', vim.o.columns)
```

Reply:
I changed it to just `2* vim.o.columns`. Reason to make it a little bigger is to prevent showing the code block hl to the right when making the screen bigger (for a split second)

Reply:
Ah, that makes sense!